src/test/resources/net/sf/jabref/importer/fileformat/BiblioscapeImporterTestUnknownFields.bib

```diff
@@ -1,18 +1,15 @@
-@article{,
-  abstract = {Protein 4.1 is an 80-kD structural component of the red blood cell (RBC) cytoskeleton.It is critical for the formation of the spectrin/actin/protein 4.1 junctional complex, the integrity of which is important for the horizontal strength and elasticity of RBCs.We and others have previously shown that multiple protein 4.1 mRNA isoforms are generated from a single genomic locus by several alternative mRNA splicing events, leading to the insertion or skipping of discrete internal sequence motifs. The physiologic significance of these motifs},
+@inbook{,
   address = {YALE UNIV DEPT INTERNAL MED NEW HAVEN,CT USA YALE UNIV DEPT PATHOL NEW HAVEN, CT USA YALE},
-  author = {Baklouti, F.; Huang, S. C.; Tang, T. K.; Delaunay, J.; Marchesi, V. T.; Benz, E. J.},
-  comment = {Tertiary Title: Regulation of splicing events within protein 4.1 pre-mRNA;Quaternary Title: Clinical medicine.;Tertiary Authors: J and Marchesi;Secondary Authors: Benz E.J. Jr.;Subject: Complexity;Quaternary Authors: Faouzi and Huang},
+  author = {Baklouti, F.},
+  booktitle = {Erythroid Differentiation},
+  comment = {Quaternary Title: Clinical medicine.},
   edition = {Second Edition},
   isbn = {0006-4971},
-  journal = {Blood},
   keywords = {Membrane skeletal protein-4.1; Insertion deletion mutations; Expression; Spectrin; Cells; Rna; Isoforms},
   language = {English},
-  note = {article},
   number = {9},
-  pages = {3934--3941},
+  pages = {3934},
```

Comment:
The thing is: If you test with real data, you should not manipulate it to contain wrong values. Why not creating a really different test file with artificial, but readable test data. For instance: `booktitle = {The Booktitle}`, `author = {Firstname Lastname}`, `pages = {2}`, etc?